Runes second series doc (#140)
rsksmart/devportal: docs/04-resources/08-runes/mockbridge/runes-deployment.md

---
sidebar_position: 3
title: Deploy Runes Mock Bridge Smart Contract

> sidebar_label: Deploy

Reviewer: ```suggestion
sidebar_label: Deploy the Mock Bridge Contract
```

Author: Same with the setup comment

tags: [rsk, rootstock, resources, tutorials,  deploy, Runes, dApps, smart contracts, Remix IDE, MetaMask]
description: "The Rootstock Runes Mock Bridge deployment page shows you how to compile and deploy runes on RemixIDE"
---

> To deploy the Runes smart contract using Remix IDE, follow these steps in detail:

Reviewer: ```suggestion
To deploy the Runes smart contract using Remix IDE, follow the steps below:
```
